client/src/context/useAuthContext.tsx

```diff
@@ -39,7 +39,13 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
       })
       .catch((error) => console.error(error));
   }, [history]);
+  /*
+  This code will automatically cause a redirect to the login page if
+  I attempt to navigate to the "/signup" route
 
+  Commenting out this functionality until it is implemented:
+  
+  '''
```

Comment:
We don't want to push up commented code. This sign up functionality actually still works, the issue lies with the `<Link />` tag. We are using Material-UIs link tag, this is good for anchor tags however, anchor tags refresh the page which triggers the auth context to redo it's logic causing the issue. 

To fix this, on the login page where it says "Register" make sure you utilize react-router-dom's Link component.

Reply:
Thank you!

I have switched to the 'react-router-dom' Link and it works like a charm. That information is invaluable - thank you!